subsys/shell/shell.c

```diff
@@ -1131,7 +1131,7 @@ static void shell_log_process(const struct shell *shell)
 		 * readable and can be used to enter further commands.
 		 */
 		if (shell->ctx->cmd_buff_len) {
-			k_sleep(K_MSEC(15));
+			k_sleep(K_MSEC(1));
```

Comment:
How come you need to do this change in the shell?

Reply:
You are right, after fixing the memory corruption caused by net-buf user-data overflow (4 bytes allocated, 10 bytes used) the calls to sleep no longer block all execution in Zephyr, I'll revert it again.